bloom-housing/bloom
# feat: rerun and release lottery modals

This PR addresses #4142


## Description

This PR adds the card content and modals an admin would see after they have run a lottery. The modals include re-running and releasing a lottery to partners.

## How Can This Be Tested/Reviewed?

On a closed listing that has a lottery, go to the lottery tab. The state in this PR is after a lottery has run. You can test that by manually updating the `lotteryLastRunAt` field to anything but null, or adding `listing.lotteryLastRunAt = new Date()` to the `lottery.tsx` file.

Ensure the main card content looks as expected, and that when you click on each button, the modals look as expected ([Re-run modal Figma](https://www.figma.com/design/PAeCVANDmGdTz2Y53ouErM/Lotteries?node-id=3345-8231&t=YJpMBWGEZbaauHjn-0), [Release modal Figma](https://www.figma.com/design/PAeCVANDmGdTz2Y53ouErM/Lotteries?node-id=3345-8745&t=YJpMBWGEZbaauHjn-0))

I confirmed the mobile experience with Em.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
